homework/js-exercises/BookList/bookList.js

```diff
@@ -0,0 +1,36 @@
+const books = [
+    {
+        title: 'The Design of Everyday Things',
+        author: 'Don Norman',
+        alreadyRead: false,
+    },
+    {
+        title: 'The Most Human Human',
+        author: 'Brian Christian',
+        alreadyRead: true,
+    },
+];
+
+let list = document.createElement('ul');
+document.body.appendChild(list);
+images = ['https://productdork.com/uploads/default/original/1X/ddbda1150301e89cdf0119d84f2a7fb8930beffb.png', 'https://images-na.ssl-images-amazon.com/images/I/51aKPFuUy1L._SX324_BO1,204,203,200_.jpg'];
+
+for (let i in books) {
+    let book = document.createElement('li');
+    let image = document.createElement('img');
+    image.setAttribute('src', images[i]);
```

Comment:
Accessing an array (images) using the iterator of another array is possible to produce breaking code.  
What if tomorrow I add another book and I forget to add an image to the array?  
The image should be added as an extra field to each book

Reply:
I thought that I shouldn't "play" with the given objects. You are right.